Contributing.md

```diff
@@ -11,4 +11,33 @@ See https://eclipse.org/legal/ECA.php for details
 
 # Eclipse
 
-The latest Buildship plugin  (>= 2.*) for Eclipse is highly recommended.
+To develop with Eclipse this is the recommended flow:
+
+1. Using the latest [Eclipse](https://www.eclipse.org/downloads/) install a Java development environment that contains these features. These are all available in the "Install New Software" under the name of the release you are using (e.g. Oxygen)
+- JDT The Java Development Tools
+- The latest Buildship plugin  (>= 2.\*) 
+  - The LSP4J build is gradle based
+- Xtend IDE
+  - Much of the code and tests are written in [Xtend](https://www.eclipse.org/xtend/)
+
+2. Fork/Clone the LSP4J repository
+
+3. Import the root of LSP4J as a Gradle project:
+  - File -> Import
+  - Existing Grade Project
+  - Select root of cloned directory as "Project root directory"
```

Comment:
`Select root of cloned directory as "Project root directory"` -> `Select the root of the cloned directory as "Project root directory"` What do you think?